src/config.rs

```diff
@@ -127,6 +128,8 @@ impl Config {
         }
 
         let prefix: PathBuf = require_env("DOCSRS_PREFIX")?;
+        let temp_dir = prefix.join("tmp");
+        fs::create_dir_all(&temp_dir)?;
```

Comment:
Doing fs modification here feels wrong, so we need to create here or can it happen later when we first is it?

Reply:
valid point, I moved it